tests/SecureServerTest.php

```diff
@@ -2,25 +2,21 @@
 
 namespace React\Tests\Socket;
 
+use React\EventLoop\LoopInterface;
+use React\Promise\Promise;
+use React\Socket\ConnectionInterface;
+use React\Socket\Connection;
```

Comment:
Nit: I think these should be switched

Reply:
Good catch, updated as suggested! :shipit: